python/samples/concepts/realtime/01b-chat_with_realtime_webrtc.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import logging
+
+from samples.concepts.realtime.utils import AudioPlayerWebRTC, AudioRecorderWebRTC, check_audio_devices
+from semantic_kernel.connectors.ai.open_ai import (
+    ListenEvents,
+    OpenAIRealtime,
+    OpenAIRealtimeExecutionSettings,
+    TurnDetection,
+)
+
+logging.basicConfig(level=logging.WARNING)
```

Comment:
Would it be helpful to add some comments about why so much logging config is required?